diablo/externals/kaltura.py

```diff
@@ -91,14 +85,11 @@ def schedule_recording(
                 # Creation date as Unix timestamp, in seconds
                 createdAt=utc_now_timestamp,
                 description=description.strip(),
-                # Duration in seconds.
-                duration=end_time.second - start_time.second,
+                # Duration in seconds
+                duration=(end_time - start_time).seconds,
                 endDate=datetime.strptime(app.config['CURRENT_TERM_END'], '%Y-%m-%d').timestamp(),
-                # Specifies the global position for the activity.
-                geoLatitude=None,
-                # Specifies the global position for the activity.
-                geoLongitude=None,
-                # Defines the intended venue for the activity.
+                geoLatitude=37.871853,
+                geoLongitude=-122.258423,
```

Comment:
![image](https://user-images.githubusercontent.com/7606442/81236863-055b2a00-8fb3-11ea-92fc-e398691cac60.png)

https://www.latlong.net/place/university-of-california-berkeley-ca-usa-11544.html